core/src/main/java/org/bitcoinj/core/listeners/ScriptsChangeEventListener.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2013 Google Inc.
```

Comment:
Copy & paste direct from the original interface (WalletEventListener.java). I can add a new copyright notice if you think it needs it, but given it doesn't make any real change apart from splitting it down, figured best to leave as-is.